osu.Game/Localisation/ResourceManagerLocalisationStore.cs

```diff
@@ -65,6 +66,11 @@ public string Get(string lookup)
                 if (manager == null)
                     return null;
 
+                // When running a debug build and in viewing english culture, use the fallbacks rather than osu-resources baked strings.
+                // This is what a developer expects to see when making changes to `xxxStrings.cs` files.
+                if (DebugUtils.IsDebugBuild && EffectiveCulture.Name == @"en")
```

Comment:
I don't know about this. I get that the fact that the resources-baked strings take precedence over fallbacks is confusing to work with, but this feels like it's moving the problem halfway through. Like the next thing we're going to get is a scenario of "oh we fixed it and it looked good in debug builds, but then we forgot to update localisations and bump translations so release is still wrong". This may not be as big deal as us missing new mods to add to osu-web every time the past few times, but it's a confusion point nevertheless.

I guess it's mostly up to you because I can't really figure out what the cadence of updating translations is, it seems essentially random - so if you're good with the fact that you have some cadence of updating that you keep to and that the translations will bubble *eventually* then maybe this is okay. But I would just as well be fine with

```suggestion
                if (EffectiveCulture.Name == @"en")
```

and with that change *all* confusion is gone and all English changes are guaranteed to propagate instantly on next build, regardless of build configuration.

Reply:
You raise a valid concern. I did consider doing this without the debug flag, but I wasn't 100% confident with the `EffectiveCulture.Name` conditional part.

@smoogipoo thoughts on always reading from fallbacks for english?

Reply:
My concern is also that we can get into a scenario where the the en resx is never pushed to crowdin for all the other localisations to be updated, _even though_ we're using the correct string in osu! (now that we're using the fallback for en).

But I'm told by @peppy that updating the resx from the strings is part of his build/deploy process, so I think this is fine.